cartscript.js

```diff
@@ -0,0 +1,77 @@
+
+const cart=document.querySelector("cart")
+const cartRemove_btns= document.querySelectorAll(".trash-btn");
+addCartRemoveEvent();
+changingItemQuantityEvent();
+itemLikeEvent();
+
+///////// deleting item from a cart
+function addCartRemoveEvent(){
+    cartRemove_btns.forEach( function(btn) {
+        btn.addEventListener("click", handleRemoveCart);
+    })
+}
+
+///// changing quantity of an item in the cart
+function changingItemQuantityEvent(){
+    let cartQuantities= document.querySelectorAll(".cart-quantity");
+    cartQuantities.forEach(function(cartQuantity){
+        cartQuantity.addEventListener("change" , handle_ChangeItemQuantity);
+    })
+}
+
+//// clicking like button or heart Icon
+function itemLikeEvent(){
+    let heartBtns = document.querySelectorAll(".like-btn");
+    heartBtns.forEach( function(heartBtn){
+       // console.log(heartBtn.value);
+        heartBtn.addEventListener("click", handle_LikeItem);
+    })
+}
+function handle_LikeItem(){
+    let c=this.className;
```

Comment:
Be careful of using the `this` keyword in JavaScript, as its value change based on the calling context. You can easily get the liked element using the passed in `event object` that is passed as an argument to any event handler function.

```js
function onLikeProduct(e) {
  const className = e.target.className;
  // rest of your code ...
}